.github/workflows/black-duck-security-scan-ci.yml

```
@@ -27,6 +27,9 @@ jobs:
       security-events: write
       actions: read
 
+    # Only run if Black Duck secrets are available
+    if: ${{ secrets.BLACKDUCKSCA_TOKEN != '' }}
```

actionlint (1.7.10)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)